xml/entity-decl.ent

```diff
@@ -241,12 +241,15 @@ suggested official replacement from rsalevsky. -->
 <!-- ++++++++++++++++++ Repository Names +++++++++++++++++++++++++ -->
 
 <!ENTITY sle_repo     "SLES12-SP2">
+<!ENTITY sle_sp3_repo "SLES12-SP3">
 <!ENTITY sleha_repo   "SLE-HA12-SP2">
 <!ENTITY ses_repo     "SUSE-Enterprise-Storage-4">
 <!ENTITY cloud_repo   "SUSE-OpenStack-Cloud-7">
+<!ENTITY cloud8_repo  "<phrase xmlns='http://docbook.org/ns/docbook'><phrase vendor='suse'>SUSE-OpenStack-Cloud-8></phrase><phrase vendor='hpe'>HPE-Helion-OpenStack-Cloud-8</phrase></phrase>">
```

Comment:
I think you can replace the cloud_repo entity.

Reply:
I wasn't sure about that, but I guess the Cloud 7 documentation is safely on another branch?

Reply:
Yes, that would be maintenance/cloud_7